examples/connectivity/plot_adhd_tangent.py

```diff
@@ -0,0 +1,105 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract and preprocess regions time series
+import nilearn.input_data
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
+    memory='nilearn_cache', memory_level=1)
+subjects = []
+for func_file in dataset.func:
+    time_series = masker.fit_transform(func_file)
+    subjects.append(time_series)
+
+# Estimate connectivity matrices
+import nilearn.connectivity
+subjects_connectivity = {}
+measures = ['tangent', 'partial correlation', 'precision', 'correlation',
+            'covariance']
+mean_connectivity = {}
+from sklearn.covariance import EmpiricalCovariance
+for measure in measures:
+    estimator = {'measure': measure, 'cov_estimator': EmpiricalCovariance()}
+    cov_embedding = nilearn.connectivity.CovEmbedding(**estimator)
+    subjects_connectivity[measure] = nilearn.connectivity.vec_to_sym(
+        cov_embedding.fit_transform(subjects))
+    # Compute the mean connectivity across all subjects
+    if measure == 'tangent':
+        mean_connectivity[measure] = cov_embedding.tangent_mean_
+    else:
+        mean_connectivity[measure] = \
+            subjects_connectivity[measure].mean(axis=0)
+
+# Plot the mean connectivity
+import numpy as np
+import nilearn.plotting
+labels = np.recfromcsv(atlas.labels)
```

Comment:
This line fails on my laptop (numpy version=1.9.1).  Is this an issue with my configuration ?